packages/lending/test/transactions/zap-supply.ts

```diff
@@ -0,0 +1,173 @@
+import { Adapter } from 'src/adapter';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import * as aaveV2 from 'src/protocols/aave-v2/tokens';
+import * as aaveV3 from 'src/protocols/aave-v3/tokens';
+import { claimToken, getBalance, snapshotAndRevertEach } from '@protocolink/test-helpers';
+import * as compoundV3 from 'src/protocols/compound-v3/tokens';
+import { expect } from 'chai';
+import hre from 'hardhat';
+import * as radiantV2 from 'src/protocols/radiant-v2/tokens';
+
+describe('Transaction: Zap Supply', function () {
+  let user: SignerWithAddress;
+  let adapter: Adapter;
+
+  const chainId = 1;
+
+  before(async function () {
+    adapter = new Adapter(chainId, hre.ethers.provider, { permitType: 'approve' });
+    [, user] = await hre.ethers.getSigners();
+
+    await claimToken(chainId, user.address, aaveV3.mainnetTokens.USDC, '2');
+    await claimToken(chainId, user.address, aaveV3.mainnetTokens.WETH, '0.4');
+  });
+
+  snapshotAndRevertEach();
+
+  context('Test Zap Supply', function () {
+    const testCases = [
+      {
+        skip: false,
+        protocolId: 'radiantv2',
+        marketId: 'mainnet',
+        params: {
+          srcToken: radiantV2.mainnetTokens.USDC,
```

Comment:
```suggestion
          srcToken: mainnetTokens.USDC,
```
suggest: get common tokens from mainnetTokens and get protocol-specific tokens from protocols

Reply:
My first thought is that importing fewer packages would be better, and we don't need to consider whether it's a common token or protocol-specific token, so I think we'll stick with the current method for now.

Reply:
The revised test cases use mainnetTokens for general tokens and use protocol.mainnetTokens for protocol specific tokens to reduce the amount of import and the length of single lines.